src/basic/cgroup-util.c

```diff
@@ -2303,8 +2303,10 @@ int cg_mask_to_string(CGroupMask mask, char **ret) {
         return 0;
```

Comment:
(comment for the patch description)

"[no one] cares for smth" means that every dislikes / has a problem with smth. I think you mean "care about" which is neutral.